examples/step-13/step-13.cc

```diff
@@ -1129,39 +1096,34 @@ namespace Step13
     // explained in several of the previous example programs, we leave it at
     // that.
     template <int dim>
-    void
-    PrimalSolver<dim>::
-    assemble_rhs (Vector<double> &rhs) const
+    void PrimalSolver<dim>::assemble_rhs(Vector<double> &rhs) const
     {
-      FEValues<dim> fe_values (*this->fe, *this->quadrature,
-                               update_values | update_quadrature_points  |
-                               update_JxW_values);
+      FEValues<dim> fe_values(*this->fe,
+                              *this->quadrature,
+                              update_values | update_quadrature_points |
+                                update_JxW_values);
 
-      const unsigned int   dofs_per_cell = this->fe->dofs_per_cell;
-      const unsigned int   n_q_points    = this->quadrature->size();
+      const unsigned int dofs_per_cell = this->fe->dofs_per_cell;
+      const unsigned int n_q_points    = this->quadrature->size();
 
-      Vector<double>       cell_rhs (dofs_per_cell);
-      std::vector<double>  rhs_values (n_q_points);
-      std::vector<types::global_dof_index> local_dof_indices (dofs_per_cell);
+      Vector<double>                       cell_rhs(dofs_per_cell);
+      std::vector<double>                  rhs_values(n_q_points);
+      std::vector<types::global_dof_index> local_dof_indices(dofs_per_cell);
 
-      typename DoFHandler<dim>::active_cell_iterator
-      cell = this->dof_handler.begin_active(),
-      endc = this->dof_handler.end();
-      for (; cell!=endc; ++cell)
+      for (auto cell : this->dof_handler.active_cell_iterators())
         {
           cell_rhs = 0;
-          fe_values.reinit (cell);
-          rhs_function->value_list (fe_values.get_quadrature_points(),
-                                    rhs_values);
-
-          for (unsigned int q_point=0; q_point<n_q_points; ++q_point)
-            for (unsigned int i=0; i<dofs_per_cell; ++i)
-              cell_rhs(i) += (fe_values.shape_value(i,q_point) *
-                              rhs_values[q_point] *
-                              fe_values.JxW(q_point));
-
-          cell->get_dof_indices (local_dof_indices);
-          for (unsigned int i=0; i<dofs_per_cell; ++i)
+          fe_values.reinit(cell);
+          rhs_function->value_list(fe_values.get_quadrature_points(),
+                                   rhs_values);
+
+          for (unsigned int q_point = 0; q_point < n_q_points; ++q_point)
+            for (unsigned int i = 0; i < dofs_per_cell; ++i)
+              cell_rhs(i) += (fe_values.shape_value(i, q_point) *
+                              rhs_values[q_point] * fe_values.JxW(q_point));
```

Comment:
Here, I agree that clang-format's output is suboptimal. Instructing not to wrap helps:
```
           for (unsigned int q_point = 0; q_point < n_q_points; ++q_point)        
             for (unsigned int i = 0; i < dofs_per_cell; ++i)                     
               cell_rhs(i) += (fe_values.shape_value(i, q_point) * //             
                               rhs_values[q_point] *               //             
                               fe_values.JxW(q_point));            //
```

Reply:
@kronbichler This is the best method of telling clang-format not to wrap lines together that I could come up with.

Reply:
My original objection to `clang-format = on/off` was on the implications on having such a statement in a tutorial program because it distracts attention. If it shows up in the online documentation, we would need to explain it - and this is simply something I do not want to spend effort on. I could live with the variant that we have it in the code base but simply drop it when generating the online documentation. That has still some educational drawback because there are statements in the code that we have not documented online, but I would not be overly bothered - because also the in-code comments are distractive in that sense. It of course also depends on how many of those we would need.

This alternative with `//` would also be ok from my point of view, so I'd be happy to open these possibilities for discussion (and let the majority decide).